test/DynamoCoreWpfTests/ViewExtensions/GraphNodeManagerViewExtensionTests.cs

```diff
@@ -22,6 +22,15 @@ namespace DynamoCoreWpfTests
 {
     public class GraphNodeManagerViewExtensionTests : DynamoTestUIBase
     {
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("VMDataBridge.dll");
+            libraries.Add("DesignScriptBuiltin.dll");
+            libraries.Add("DSCoreNodes.dll");
+            base.GetLibrariesToPreload(libraries);
+        }
```

Comment:
these are needed by some of the graphs opened during GraphNodeManagerViewExtensionTests  tests